src/main/java/com/yurjinia/project_structure/column/service/ColumnService.java

```diff
@@ -89,15 +95,27 @@ public void deleteColumn(String projectCode, String boardCode, String columnName
     }
 
     public ColumnEntity getColumnByName(String projectCode, String boardCode, String columnName) {
-        return boardService.getBoard(boardCode, projectCode).getColumns().stream()
+        BoardEntity boardEntity = boardService.getBoard(boardCode, projectCode);
+
+        return boardEntity.getColumns().stream()
```

Comment:
make it as one stream without creating a separate variable

Reply:
It is so unwilling to work

Reply:
return boardService.getBoard(boardCode, projectCode).getColumns().stream()
               .filter(columnEntity -> columnEntity.getName().equals(columnName)).findFirst()
                .orElseThrow(() -> new CommonException(ErrorCode.COLUMN_NOT_FOUND, HttpStatus.NOT_FOUND));